cmd/cloudAccount_show.go

```diff
@@ -77,13 +77,14 @@ func (t *cloudShowCmd) run() error {
 	util.PrintResult(
 		t.out,
 		cloud,
-		[]string{"ID", "Name", "TeamID", "AccountID", "Arn"},
+		[]string{"ID", "Name", "TeamID", "AccountID", "Arn", "IsValid"},
```

Comment:
default will be false for cloudaccounts which dont have isValid property on them.